plugin/assets/disable-tooltips.js

```diff
@@ -6,6 +6,7 @@ jQuery(document).ready(() => {
   const nux = select('core/nux');
 
   if (!nux) {
+    // needed for WP version <=6.0
```

Comment:
Is this statement 100% correct? I'm not sure when `nux` was introduced but was only recently removed. If unsure, let's just remove the comment.

Reply:
I tested it the fix without that line on a few WordPress versions and the fix still worked on WP6.1 and WP6.2 but didn't work on WP6.0.  So that's why I put that code comment there.  I can remove it if you like.

Reply:
Ok, I made the comment more vague so we can be sure it's accurate.